From af1a9629d994e3776713d2c77591e9ce561bc282 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Wed, 16 Oct 2019 11:43:57 -0700
Subject: [PATCH] Fix lib64

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 50c90ce..83f48f2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -186,7 +186,7 @@ if (NOT LUA)
       set(INSTALL_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib/lua/${LUA_VERSION_MAJOR}.${LUA_VERSION_MINOR}")
     endif (WIN32)
   else (BUILD_MODULE)
-    set(INSTALL_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib"
+    set(INSTALL_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib64"
       CACHE PATH "Installation directory for libraries")
     set(INSTALL_INC_DIR "${CMAKE_INSTALL_PREFIX}/include/luv"
       CACHE PATH "Installation directory for headers")
-- 
2.23.0

